### ISSUE TITLE
datetime-sb-0.2.3 can't compile its test suite

### ISSUE THREAD

**Issue by peti** [stackbuilders/datetime#1]
Citing from http://hydra.cryp.to/build/900236/log/raw:

```
Preprocessing test suite 'test' for datetime-sb-0.2.3...

test/Suite.hs:5:18:
    Could not find module ‘Data.DateTimeTest’
    Perhaps you meant
      Data.DateTime (from datetime-sb-0.2.3@datet_0lpeSY1f3Lz6LTf8WQswws)
    Use -v to see a list of the files searched for.
```

**Comment by jsl** [comment]
Hi @peti, thanks for reporting this issue. It seems that the distribution on hackage was missing a test file. I've fixed this issue and uploaded version 0.2.4 to hackage. Can you let me know if that resolves the problem?

**Comment by peti** [comment]
The new version compiles fine http://hydra.cryp.to/job/nixpkgs/haskell-updates/haskellPackages.datetime-sb.x86_64-linux. Thank you very much for the quick update!
